StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetConfigurationTests.swift

```diff
@@ -60,4 +60,28 @@ class PaymentSheetConfigurationTests: XCTestCase {
         configuration.billingDetailsCollectionConfiguration.phone = .always
         XCTAssert(configuration.isUsingBillingAddressCollection())
     }
+
+    func testSTPPaymentMethodBillingDetailsToPaymentSheetBililngDetails() {
+        var billingDetails = STPPaymentMethodBillingDetails()
+        billingDetails.name = "Jane Doe"
+        billingDetails.email = "janedoe@test.com"
+        billingDetails.phone = "+18885551234"
+        billingDetails.address = STPPaymentMethodAddress()
+        billingDetails.address?.line1 = "123 Main Street"
+        billingDetails.address?.line2 = ""
+        billingDetails.address?.city = "San Francisco"
+        billingDetails.address?.state = "California"
+        billingDetails.address?.country = "US"
+
+        let psBillingDetails = billingDetails.toPaymentSheetBillingDetails()
+
+        XCTAssertEqual(psBillingDetails.name, "Jane Doe")
+        XCTAssertEqual(psBillingDetails.email, "janedoe@test.com")
+        XCTAssertEqual(psBillingDetails.phone, "+18885551234")
```

Comment:
This field's docstring states formatting requirements we should consider preserving.  Alternatively, if it does make sense to include the +, we should update the docstring.
> /// The customer's phone number without formatting (e.g. 5551234567)

Reply:
Resolved offline. Introducing `phoneNumberForDisplay` & updated docstring to resolve.